### PR TITLE
docs(homebrew): adjust details about installation

### PR DESCRIPTION
**Pull request by lewis-yeung**
### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines

### Description

This may help to prevent other Homebrew users from getting the same confusion as I got in #2023. ~~The Way **#B** to install OMP could be more friendly to non-Gophers.~~

**P.S.** I had ever thought that _**homebrew-oh-my-posh**_ could provide a way of installing from binary releases (e.g., [go-task](https://github.com/go-task/homebrew-tap/blob/master/Formula/go-task.rb), which uses [GoReleaser](https://github.com/goreleaser/goreleaser)) since this could be more friendly to non-Gophers. However, _**homebrew-core**_ already provides the way of installing from bottles, so now I think it appears to be non-essential.

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: docs/docs/install-homebrew.mdx [hunk 1]
```diff
@@ -1,17 +1,25 @@
 A [Homebrew][brew] formula is available for easy installation. When installing Homebrew for Linux, be sure to follow *[Next steps][nextsteps]* instructions to add Homebrew to your PATH and to your bash shell profile script.
 
+There are two different ways to install Oh My Posh via Homebrew:
+
+- **#A**
+
 ```bash
-brew tap jandedobbeleer/oh-my-posh
-brew install oh-my-posh
+brew install jandedobbeleer/oh-my-posh/oh-my-posh
```
Comment (JanDeDobbeleer) [inline docs/docs/install-homebrew.mdx:5]: I would only add this line, 2 ways of installing where one is one that I don't maintain is not something I wish to support ;-). The fact that it also installs golang is irrelevant in my book as that's how brew works. That is completely unrelated to oh-my-posh and goes for any installation.

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]
see in-line comment

File: docs/docs/install-homebrew.mdx (hunk 1 above)
Comment (lewis-yeung) [reply docs/docs/install-homebrew.mdx:5]: @JanDeDobbeleer Got your thought. Removed others while the executable path should be `$(brew --prefix)/bin` indeed.

**Comment by lewis-yeung** [comment]
@all-contributors Please add @lewis-yeung for code and doc.
